src/main/scala/thurloe/service/StatusService.scala

```diff
@@ -0,0 +1,32 @@
+package thurloe.service
+
+import spray.http.StatusCodes
+import spray.routing.HttpService
+import thurloe.database.DataAccess
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{Failure, Success}
+
+trait StatusService extends HttpService {
+
+  val dataAccess: DataAccess
+  val getStatus = path("status") {
+    get {
+      onComplete(dataAccess.status()) {
+        case Success(_) =>
+          respondWithStatus(StatusCodes.OK) {
+            complete {
+              ""
```

Comment:
I think we should return valid json:
{"status": "up"}
{"status": "down", "error": "message"}